platform-configs/protectli-v1210.robot

```diff
@@ -10,3 +10,8 @@ ${CPU_MIN_FREQUENCY}=           800
 ${PLATFORM_CPU_SPEED}=          2.00
 ${PLATFORM_RAM_SPEED}=          2933
 ${PLATFORM_RAM_SIZE}=           4096
+
+# List of ethernet interfaces
+@{ETH_PORTS}=                   64-62-66-2f-00-12
+...                             64-62-66-2f-00-13
+${ETHERNET_ID}=                 [8086:125c]
```

Comment:
```suggestion
${ETHERNET_ID}=                 8086:125c
```

And everywhere else.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/515/commits/efd955f8de238bd905c88dd7c8d242ea1beb15dd